doc/release-process.md

```diff
@@ -12,10 +12,35 @@ It is best if the maintainers are present during the release, so they can help e
 
 This process also assumes that there will be no minor releases for old major releases.
 
+We aim to cut a regular release every 3-4 months, approximately twice as frequent as major Bitcoin Core releases. Every second release should be published one month before the feature freeze of the next major Bitcoin Core release, allowing sufficient time to update the library in Core.
+
+## Sanity Checks
+Perform these checks before creating a release:
+
+1. Ensure `make distcheck` doesn't fail.
+```shell
+./autogen.sh && ./configure --enable-dev-mode && make distcheck
+```
+2. Check installation with autotools:
+```shell
+dir=$(mktemp -d)
+./autogen.sh && ./configure --prefix=$dir && make install && ls -l $dir/include $dir/lib
```

Comment:
Maybe use `git clean -xdff` to start from the clean repo?

Reply:
> Maybe use `git clean -xdff` to start from the clean repo?

I think this will potentially kill some files that people still need (the same reason why @jonasnick didn't like `rm -rf build`. We could start in a fresh clone or worktree, but that may be overkill.

Reply:
I hope that this will solve itself once it runs as part of CI.

Reply:
Added `make install`